Tests/XCUITests/PrivateBrowsingTest.swift

```diff
@@ -52,7 +52,8 @@ class PrivateBrowsingTest: BaseTestCase {
         waitForTabsButton()
         navigator.goto(TabTray)
 
-        waitForExistence(app.cells.staticTexts[url2Label])
+        waitForExistence(app.otherElements["Tabs Tray"])
+        XCTAssertNotNil(app.otherElements["Tabs Tray"].collectionViews.cells.staticTexts.element(boundBy: 1).label.range(of: url2Label))
```

Comment:
How about something like this?

Reply:
Yes, we could try something like that... thanks for the suggestion Clare